Documentation/MigrateToComposer/Requirements.rst

```diff
@@ -44,9 +46,31 @@ accessible via HTTP request.
     └── typo3temp/
 
 You will need a web root folder in your project. You can find many
-tutorials with different names for your web root folder. The truth is:
-the name does not matter because we can configure it in the settings in
-a later step. We will use :file:`public/` in our example.
+tutorials with different names for your web root folder (e.g. :file:`www/`,
+:file:`htdocs/`, :file:`httpdocs/`, :file:`wwwroot/`, :file:`html/`).
+The truth is: the name does not matter because we can configure it in the
+settings in a later step. We will use :file:`public/` in our example.
+
+**Also bad:**
```

Comment:
Why also. Additionally, why is the example bad when "public" is the web root? ;-)

Personally, I would remove that. Just tell the good story.